app/script/view_model/list/StartUIViewModel.js

```diff
@@ -140,23 +140,22 @@ z.ViewModel.list.StartUIViewModel = class StartUIViewModel {
       );
     });
 
-    this.enableIntegrations = ko.pureComputed(
-      () => true || (this.is_team() && !z.util.Environment.frontend.is_production())
-    );
+    this.enableIntegrations = this.integrationRepository.enableIntegrations;
 
-    this.show_content = ko.pureComputed(
-      () => this.show_contacts() || this.show_matches() || this.show_search_results()
-    );
+    this.show_content = ko.pureComputed(() => {
```

Comment:
why not the implicit return?

Reply:
Because prettier already turns it into a multiline thing. If code does not make the one line cutoff, I prefer this code style.